CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# global owners. 
+* @tristen @samanpwbb
```

Comment:
I've usually seen the [weird indentation](https://help.github.com/en/articles/about-code-owners#codeowners-syntax) after the `*` but I'm sure this still works